firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1302,67 +1241,7 @@ class BrowserViewController: UIViewController,
         addressToolbarContainer.isHidden = true
     }
 
-    private func switchToolbarIfNeeded() {
-        var updateNeeded = false
-
-        // FXIOS-10210 Temporary to support updating the Unified Search feature flag during runtime
```

Comment:
Just to double check: Is there other clean up needed related to this task? https://mozilla-hub.atlassian.net/browse/FXIOS-10210

Reply:
Not for now as unified search is not enabled yet.